lib/client.js

```diff
@@ -120,17 +126,52 @@ exports.getHistory = async ({
   return res.data;
 };
 
+const _signRequest = async ({keyId, url, json}) => {
+  const created = Math.floor(Date.now() / 1000);
+  const includeHeaders = [
+    '(created)',
+    // (key-id) will include the peerId in the signature itself
+    '(key-id)',
+    '(request-target)',
```

Comment:
Should have `(expires)` and `host` as well. Expiration can be 5 minutes into the future.

Reply:
addressed here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/249/commits/8d345847d6fe7256b303da8a4b1074a2979696d3